### DISCUSSION TITLE
WallarGe atomic clock temperature sensor not picked up

### DISCUSSION THREAD

**Discussion by lumpynose** [merbanan/rtl_433#3007] [Q&A]
On the back of the sensor it says "Model name:  CLTX001" and "FCC ID: 2AYIQ-TX100".

The amazon product link is [here](https://www.amazon.com/WallarGe-Atomic-Indoor-Outdoor-Temperature/dp/B0BYNJW532?th=1).

I'm not getting anything for it with either the rtl_433 that Debian provides or by compiling and installing the one here on github.

My rtl is the one from rtl-sdr.com, version 1 I guess.

The startup messages also included

```
Found Rafael Micro R820T tuner
SDR: Using device 0: Realtek, RTL2838UHIDIR, SN: 00000001, "Generic RTL2832U OEM"
Exact sample rate is: 250000.000414 Hz
```

I have two other 433 MHz temperature sensors which work and have been working well for several years.

**Comment by zuckschwerdt** [comment]
Try to locate the signal and then grab a sample. https://triq.org/rtl_433/ANALYZE.html
Verify in https://triq.org/pdv/ and upload as zip here.

**Reply by knarrff** [reply]
This. Also, the FCC report might help: https://fcc.report/FCC-ID/2AYIQ-TX100/5074204.pdf

**Comment by DennisKehrig** [comment]
PR is out: #3454
